docs/content/users/install/ddev-installation.md

```diff
@@ -48,6 +48,8 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
     mkcert -install
     ```
 
+Note: Some versions of Firefox (Developer Edition, Flatpaks) may need some [extra work](https://github.com/FiloSottile/mkcert/issues/370#issuecomment-1280377305), see also [https://github.com/ddev/ddev/issues/5415](this issue).
```

Comment:
I’m not personally a fan of `Note:` because we have admonitions available for emphasis but this reads fine without it, so I’d remove it. Second link also needs fixing up:

```suggestion
Some versions of Firefox (Developer Edition, Flatpaks) may need some [extra work](https://github.com/FiloSottile/mkcert/issues/370#issuecomment-1280377305), see also [this issue](https://github.com/ddev/ddev/issues/5415).
```

Reply:
Thanks for your suggestions, and for fixing the link. Committed!